lib/state/simperium/middleware.ts

```diff
@@ -351,13 +351,23 @@ export const initSimperium = (
         return result;
       }
 
+      case 'RESTORE_NOTE_REVISION': {
+        action.note.tags.map(t).forEach((tagHash) => {
+          if (!prevState.data.tags.has(tagHash)) {
+            queueTagUpdate(tagHash, 10);
+          }
+        });
```

Comment:
this could resolve a lot of issues we currently have with tags not syncing properly